src/mahoji/commands/runecraft.ts

```diff
@@ -131,6 +133,17 @@ export const runecraftCommand: OSBMahojiCommand = {
 			boosts.push(ringStr);
 		}
 
+		const amuletId = Inventions.find(inv => inv.name === 'Abyssal amulet')!.item.id;
```

Comment:
Prefer `inv.id === InventionID.AbyssalAmulet`, which makes this typesafe (wouldnt throw an error if we changed the item name)